app/scripts/modules/core/ci/jenkins/igor.service.js

```diff
@@ -9,19 +9,19 @@ module.exports = angular.module('spinnaker.core.ci.jenkins.igor.service', [
   .factory('igorService', function (settings, Restangular) {
 
     function listMasters() {
-      return Restangular.one('builds').getList();
+      return Restangular.one('v2').one('builds').getList();
     }
 
     function listJobsForMaster(master) {
-      return Restangular.one('builds', master).all('jobs').getList();
+      return Restangular.one('v2').one('builds', master).all('jobs').getList();
     }
 
     function listBuildsForJob(master, job) {
-      return Restangular.one('builds', master).one('jobs', job).all('builds').getList();
+      return Restangular.one('v2').one('builds', master).one('builds').one('jobs', job).getList();
```

Comment:
this url seems to be incorrect, it adds an additional /jobs/ 

http://192.168.99.100:8084/v2/builds/spinnaker/builds/jobs/SPINNAKER-package-igor

http://192.168.99.100:8084/v2/builds/spinnaker/builds/SPINNAKER-package-igor